internal/pkg/service/buffer/store/file_test.go

```diff
@@ -168,53 +167,6 @@ file/<STATE>/00001000/my-receiver/my-export/2006-01-01T08:04:05.000Z
 	}
 }
 
-func TestStore_ListFilesInStateWithEnd(t *testing.T) {
-	t.Parallel()
-	ctx := context.Background()
-	store := newStoreForTest(t)
-
-	receiverKey := key.ReceiverKey{ProjectID: 1000, ReceiverID: "my-receiver"}
-	exportKey := key.ExportKey{ExportID: "my-export", ReceiverKey: receiverKey}
```

Comment:
Tento testo pouzival Jakub ked to vyvijal.
Je duplicitny, `WithEnd` ma svoj test, a `cleanup` ma tiez svoj test.